--- a/compiler/src/iree/compiler/Codegen/LLVMGPU/KernelConfig.cpp
+++ b/compiler/src/iree/compiler/Codegen/LLVMGPU/KernelConfig.cpp
@@ -1796,7 +1796,7 @@
 // Sort Pipeline Configuration
 //====---------------------------------------------------------------------===//
 
-static LogicalResult setSortConfig(IREE::GPU::TargetAttr target,
+static LogicalResult setSortConfigDistribute(IREE::GPU::TargetAttr target,
                                    mlir::FunctionOpInterface entryPoint,
                                    Operation *op) {
   MLIRContext *context = op->getContext();
@@ -2498,7 +2498,7 @@
       })
       .Case([&](IREE::LinalgExt::SortOp sortOp) {
         LDBG() << "Sort Config";
-        return IREE::GPU::setSortConfig(target, entryPointFn, sortOp);
+        return setSortConfigDistribute(target, entryPointFn, sortOp);
       })
       .Case([&](IREE::LinalgExt::ArgCompareOp argCompareOp) {
         LDBG() << "ArgCompare Config";
